server/routes/ticket.py

```
@@ -0,0 +1,307 @@
+                client = User.query.filter_by(id=client_id, role=Role.CLIENT).first()
+                if not client:
+                    return restful_response(
+                        status="error", message="Invalid client", status_code=400
+                    )
+
+                if admin_id is None:
+                    return restful_response(
+                        status="error",
+                        message="admin_id is required",
+                        status_code=400,
+                )
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)